graphql/concrete_notification/fields.ts

```diff
@@ -99,3 +101,53 @@ export class ExtendedFieldsConcreteNotificationResolver {
         return Object.values(byCampaign);
     }
 }
+
+@ObjectType()
+class ConcreteNotificationFormatted {
+    @Field((type) => ID)
+    id: number;
+    @Field()
+    headline: string;
+    @Field()
+    body: string;
+    @Field()
+    notificationClass: string;
+    @Field() // is there a better solution than "string" as type?
+    createdAt: string;
+}
+
+@ObjectType()
+class NotificationsList {
+    @Field((type) => [ConcreteNotificationFormatted])
+    notifications: ConcreteNotificationFormatted[];
+    @Field()
+    user: string; // for debugging, @TODO: remove later
+}
+
+const dummyMinutes = [0, 5, 10, 15, 20, 30, 60, 120, 150, 60 * 24, 60 * 48];
+const getDummyNotifications = () =>
```

Comment:
If test data is needed, this should be directly filled into the database (table concrete_notification / notification) in web/dev.ts.

Reply:
This ticket [LF-6](https://typedigital.atlassian.net/browse/LF-6?atlOrigin=eyJpIjoiOWI2MGNiYTcxMDIyNDExZjk1NzI4MTJmOWMwN2FkN2UiLCJwIjoiaiJ9) is an intermediate step with the goal of display something in frontend and establishing the connection to graphql. Next ticket will be more refined and we would have more clarity on what data is needed. I would suggest to move the test data to the database with the next ticket (probably [LF-7](https://typedigital.atlassian.net/browse/LF-7?atlOrigin=eyJpIjoiZmIzZjBjNGNlZGJjNDY3MzhkN2QzNzRlMmQ2MjllZGIiLCJwIjoiaiJ9) or a future ticket introducing templates)